x-pack/solutions/observability/plugins/apm/public/components/app/metrics/static_dashboard/dashboards/dashboard_catalog.ts

```diff
@@ -71,6 +73,18 @@ export async function loadDashboardFile(filename: string) {
         './opentelemetry_dotnet.json'
       );
     }
+    case 'classic_apm-edot-python': {
+      return import(
+        /* webpackChunkName: "lazyPythonOtelDashboard" */
+        './opentelemetry_python.json'
```

Comment:
Sorry I missed that PR: 
- Do we want to show the same dashboard for `classic_apm-edot-python` **and** `otel_native-edot-python`?
- Dashboard name should match the pattern: so we should have `classic_apm-edot-python.json` and  `otel_native-edot-python.json` - the other dashboards should be renamed too - the reason they are not renamed is that I mapped the existing files and I would suggest following the new naming convention in the future cc: @AlexanderWert

Reply:
@jennypavlova will fix the chunk names, my thought was that since they are the same file it would have been fine to reuse it.
This dashboard uses a field that is queryable with same name in both an instance ingesting data with the collector and one ingesting with the apm-server.

Reply:
@xrmx Thanks, I thought they were different  - that was why this logic was used to build the dashboard file name. It will be easier if we stick to them as we could potentially have a different one for `opentelemetry` python (for example: `otel_native-otel_other-python`) and it could be misleading. I am not sure if this is the case with python but would be nice to stick to the same names to avoid mistakes when mapping.

Reply:
@jennypavlova Are you ok with `./common-edot-python.json` to signal it is shared? Would prefer to not duplicate it or add links if possible.

Reply:
@jennypavlova PR here, for now I've decided to reduce the scope and just keep the otel native one with a file named as the matched triplet https://github.com/elastic/kibana/pull/215005